testing/e2e/clients/acr.go

```diff
@@ -118,12 +120,12 @@ func (a *acr) BuildAndPush(ctx context.Context, imageName, dockerfilePath, docke
 			// We've tried alternate strategies like polling the sdk to see if the acr exists but that
 			// tells us it exists then the acr command fails. This is the only reliable way we've found
 			// to wait for the acr to be ready.
-			if (!cantFindAcrRegex.Match(errLog.Bytes())) && (!throttledRegex.Match(errLog.Bytes())) {
+			if (!cantFindAcrRegex.Match(errLog.Bytes())) && (!throttledRegex.Match(errLog.Bytes())) && (!hostRegex.Match(errLog.Bytes())) {
 				lgr.Error("failed to build and push acr image")
 				return fmt.Errorf("running build and push command: %w", err)
 			}
 
-			if time.Since(start) > 1*time.Minute {
+			if time.Since(start) > 5*time.Minute {
```

Comment:
why is this bumped to 5 minutes? just fail faster

Reply:
if we have to bump this because the host regex error happens after building write custom logic for that. wait a different amount of time when getting the hostregex error vs finding regex or throttled

Reply:
reverted to failing fast